test/misc.jl

```diff
@@ -452,3 +452,18 @@ if is_windows()
         err18083 == 0 && error(Libc.GetLastError())
     end
 end
+
+let
+    old_have_color = Base.have_color
+    try
+        eval(Base, :(have_color = true))
```

Comment:
this was fine for now, better to test things. we can think about making color not a global elsewhere.

hopefully can restore these from reflog or old commits?